CMakeLists.txt

```diff
@@ -1,94 +1,48 @@
-cmake_minimum_required(VERSION 2.8.12)
+###########################################################################
+#                                                                         #
+# Note: The bulk of the build system is located in the cmake/ directory.  #
```

Comment:
I think this comment is not correct anymore with this PR merged

Reply:
copy-paste issue from opm-output. improved.